src/types.ts

```diff
@@ -72,3 +72,105 @@ export type Breadcrumbs = {
 };
 
 export type OrderedBreadcrumbs = Array<EnrichedCrumb>;
+
+export interface Passport {
+  data?: Record<string, any>;
+}
+
+// https://docs.payments.service.gov.uk/api_reference/#status-and-finished
+export enum PaymentStatus {
+  created = "created",
+  started = "started",
+  submitted = "submitted",
+  capturable = "capturable",
+  success = "success",
+  failed = "failed",
+  cancelled = "cancelled",
+  error = "error",
+  unknown = "unknown", // used when response status is not valid
+}
+
+// https://docs.payments.service.gov.uk/making_payments/#receiving-the-api-response
+export interface GovUKPayment {
+  amount: number;
+  reference: string;
+  state: {
+    status: PaymentStatus;
+    finished: boolean;
+  };
+  payment_id: string;
+  created_date: string;
+  _links: {
```

Comment:
nit: is `_links` necessary to include in this interface? I think this is only used in the editor during a live session but not after that.

Reply:
This is a fair question - we'll need to think about where and how we want to handle types across the repos. I've kept it in full here for now for simplicity but agree that this part of the model is an implementation concern.